js/components/urlBar.js

```diff
@@ -250,9 +256,13 @@ class UrlBar extends ImmutableComponent {
   }
 
   onActiveFrameStop () {
-    this.restore()
-    windowActions.setUrlBarSelected(true)
-    windowActions.setUrlBarActive(false)
+    if (this.isFocused()) {
+      windowActions.setUrlBarActive(false)
+      if (!this.shouldRenderUrlBarSuggestions) {
+        this.restore()
+        windowActions.setUrlBarSelected(true)
+      }
```

Comment:
I think there's some kind of a rebase issue here, please rebase from current master